packages/mui-base/src/Slider/Root/SliderContext.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+import * as React from 'react';
+import type { SliderRoot } from './SliderRoot';
+
+/**
+ * @ignore - internal component.
+ */
+export const SliderContext = React.createContext<SliderRoot.Context | undefined>(undefined);
```

Comment:
Context type should be defined in this file (and be called `SliderContext`)

Reply:
Updated ~ I've also renamed it to `SliderRootContext`/`useSliderRootContext` to match other components

Reply:
Yeah, that's actually what I meant 😅